test/02-mint.test.js

```diff
@@ -0,0 +1,59 @@
+const MyToken = artifacts.require("MyToken");
+const Assert = require('truffle-assertions');
+
+contract("Mint", accounts => {
+
+    let contractInstance;
+    const decimals = 6;
+    const ownerAddress = accounts[0];
+    const address1 = accounts[1];
+
+    beforeEach(async () => {
+        contractInstance = await MyToken.new();
+    });
+
+    it('mint should throw if contract is paused', async () => {
+        const mintValue = 1000;
+        
+        await contractInstance.pause({ from: ownerAddress });
+        
+        await Assert.reverts(
+            contractInstance.mint(address1, mintValue, { from: ownerAddress }),
+            'Pausable: paused'
+        );
+    });
+
+    it('mint should throw if to address is invalid', async () => {
+        await Assert.reverts(
+            contractInstance.mint('0x0000000000000000000000000000000000000000', 1000, { from: ownerAddress }),
+            'ERC20: to address is not valid'
+        );
+    });
```

Comment:
also as above
0x0000000000000000000000000000000000000000  - is ZERO_ADDRESS. use constants from openzepellin-test-helpers